docs/running/notebooks/01_running_program.ipynb

```diff
@@ -35,27 +35,26 @@
     "\n",
     "This program creates a two-qubit quantum circuit that prepares a Bell state, measures the result, and prints the measured probability distribution.\n",
     "\n",
-    "### Running the Program\n",
+    "### Running the program\n",
     "\n",
-    "To run the program, we need to import the necessary classes and configure them. One of these classes is QuantumServerless, which is a client class for interacting with compute resources.\n",
+    "To run the program, we need to import the necessary classes and configure them. One of these classes is `QuantumServerless`, which is a client class for interacting with compute resources.\n",
     "\n",
-    "QuantumServerless takes a Provider object as a constructor argument. The Provider object stores configuration information about our compute resources, such as where they are located and how to connect to them. In this example, we will use a provider that is connected to a local Docker Compose setup. In this case, it allows us to run the program locally on our machine. If you want to run the program elsewhere, you will need to provide the corresponding host and authentication details."
+    "`QuantumServerless` takes a `Provider` object as a constructor argument. The `Provider` object stores configuration information about our compute resources, such as where they are located and how to connect to them. In this example, we will use a provider that is connected to a local Docker Compose setup. In this case, it allows us to run the program locally on our machine. If you want to run the program elsewhere, you will need to provide the corresponding host and authentication details."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
    "id": "81dd7807-7180-4b87-bbf9-832b7cf29d69",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from quantum_serverless import QuantumServerless, GatewayProvider\n",
-    "import os"
```

Comment:
we need the `import os` for getting the host envvar in the next code block

Reply:
Thanks, I didn't mean for my changes to automatically override your conflicting changes in main. I assumed it would let me resolve them manually.